http/interceptor.go

```diff
@@ -87,16 +88,16 @@ func wrap(w http.ResponseWriter, r *http.Request, tx types.Transaction) (
 		// as body hasn't being analized yet.
 		if tx.IsInterrupted() {
 			// phase 4 interruption stops execution
-			w.WriteHeader(i.statusCode)
+			i.w.WriteHeader(i.statusCode)
```

Comment:
Is this change needed?

Reply:
Added a response [here](https://github.com/corazawaf/coraza/pull/539#discussion_r1073588556), it is just a matter of making the calls to the same function homogeneous. I will revert if it does not seem right